docker-files/alpine.Dockerfile

```diff
@@ -0,0 +1,78 @@
+#
+# Glances Dockerfile (based on Alpine)
+#
+# https://github.com/nicolargo/glances
+#
+
+FROM alpine:3.13 as build
```

Comment:
Do you think that it is necessary to fix the Alpine version inthe DockerFile ?

Reply:
Maybe don't exactly the alpine version, but somehow I wanted to stick with a known python version for the package copies between the stages in the Dockerfiles.
Without it the packages won't get copied on a python minor upgrade, since the folder structure is changing.